src/main/scala/loamstream/drm/lsf/BacctAccountingClient.scala

```diff
@@ -50,9 +50,22 @@ final class BacctAccountingClient(
      * Documentation on bacct's output format:
      * https://www.ibm.com/support/knowledgecenter/en/SSWRJV_10.1.0/lsf_command_ref/bacct.1.html
      */
-    val node = rawBacctOutput.collectFirst { case Regexes.node(n) => n }
+    def stripLineEndings(s: String): String = s.replaceAll("\\n", "").replaceAll("\\r", "")
```

Comment:
Not sure what the output looks like, but would it be better to replace `\n` with a space so words/data are potentially joined together unintentionally?

Reply:
>`"\\r?\\n"` 

Thanks, I'll do that.

> would it be better to replace \n with a space so words/data are potentially joined together unintentionally?

I added a comment in another commit which explains this more, but due to `bacct`'s new output format, this code needs to join words together.  The long version: `bacct` 9.x, which this code was written against, had a flag that allowed disabling some/most line-wrapping in its output.  `bacct` 10.x (what EBI now runs) removed that option, and so output that used to look like this:
```
  blah blah blah <a field delimited by brackets> la la la
```
now looks like
```
  blah blah b
     lah <a fie
     ld delimit
     ed by bra
     ckets> la 
     la la
```
I've exaggerated the small width of the lines for effect, but note the line breaks and indenting.  This code needs to be able to get out the stuff in between, say, the `<>`s without the line breaks and extra spaces.  Yes, it's lame. :\